source/dataman/CLS/CLSDbUpgrade1Pt1.cpp

```diff
@@ -0,0 +1,128 @@
+#include "CLSDbUpgrade1Pt1.h"
+
+#include "util/AMErrorMonitor.h"
+
+CLSDbUpgrade1Pt1::CLSDbUpgrade1Pt1(CLSAppController::CLSBeamlineID targetFacilityId, const QString &databaseNameToUpgrade, QObject *parent)
+	: AMDbUpgrade(databaseNameToUpgrade, parent)
+{
+	targetFacilityId_ = targetFacilityId;
+}
+
+CLSDbUpgrade1Pt1::~CLSDbUpgrade1Pt1()
+{
+
+}
+
+QStringList CLSDbUpgrade1Pt1::upgradeFromTags() const
+{
+	return QStringList();
```

Comment:
empty string list for `upgradeFromTags()` means this upgrade doesn't have dependencies.

Some upgrades have dependencies, such as some old version upgrades. Then we should list the dependencies here.